develop branch dose not build

```Compiling DistrhoUIMain.cpp (LV2)
In file included from ../../dpf/distrho/DistrhoUIMain.cpp:26:
../../dpf/distrho/src/DistrhoUILV2.cpp: In member function ‘void DISTRHO::UiLv2::sendNote(uint8_t, uint8_t, uint8_t)’:
../../dpf/distrho/src/DistrhoUILV2.cpp:318:35: error: ‘fMidiEventURID’ was not declared in this scope
  318 |         atomMidiEvent.atom.type = fMidiEventURID;
      |                                   ^~~~~~~~~~~~~~
make[1]: *** [../../dpf/Makefile.plugins.mk:163: ../../build/Nekobi/DistrhoUIMain_LV2.cpp.o] Error 1
make[1]: Leaving directory '/home/marbangens/DPF-Plugins/plugins/Nekobi'
make: *** [Makefile:47: plugins] Error 2
```

--- a/distrho/src/DistrhoUILV2.cpp
+++ b/distrho/src/DistrhoUILV2.cpp
@@ -315,7 +315,7 @@ protected:
 
         LV2_Atom_MidiEvent atomMidiEvent;
         atomMidiEvent.atom.size = 3;
-        atomMidiEvent.atom.type = fMidiEventURID;
+        atomMidiEvent.atom.type = fURIDs.midiEvent;
 
         atomMidiEvent.data[0] = channel + (velocity != 0 ? 0x90 : 0x80);
         atomMidiEvent.data[1] = note;

